Src/library/net/Clib/network.c

```diff
@@ -29,12 +29,15 @@ indexing
 #define FD_SETSIZE 256
 #include <winsock2.h>
 #include <ws2tcpip.h>
+#include <Mswsock.h>
```

Comment:
Note that if I remove the include of `MSWSock.h`, and change the code to be
```
EIF_INTEGER c_sendfile_noexception(EIF_INTEGER out_fd, FILE* f, EIF_INTEGER length)
	/* transmission of file content from file `f` of size length through socket out_fd.
	 * if `length` is 0, then send the whole file.
	 *	NO exception is raised, and eventual error is return as result!
	 */
{
#ifdef EIF_WINDOWS
	HMODULE dllHandle;
	BOOL (PASCAL * _TransmitFile)(SOCKET, HANDLE, DWORD, DWORD, LPOVERLAPPED, void*, DWORD) = NULL;
	
	dllHandle = LoadLibrary("Mswsock.dll");
	if (dllHandle != NULL) {
		_TransmitFile = (BOOL (PASCAL *) (SOCKET, HANDLE, DWORD, DWORD, LPOVERLAPPED, void*, DWORD))GetProcAddress(dllHandle, "TransmitFile");
	}
	if (_TransmitFile==NULL) {
		return (EIF_INTEGER) c_unoptimized_sendfile_noexception(out_fd, f, length);
	} else {
		BOOL retval;
		HANDLE hFile = (HANDLE)_get_osfhandle(fileno(f));
		OVERLAPPED ovlp;
		memset(&ovlp, 0, sizeof(OVERLAPPED));
		retval = _TransmitFile((EIF_SOCKET_TYPE) out_fd, (HANDLE) hFile, (DWORD) length, 
					(DWORD) 0, (LPOVERLAPPED) &ovlp, (void*) NULL,(DWORD) 0x20 /*TF_USE_KERNEL_APC*/);
		if (retval) {
				/* Fix me, find a way to get the number of bytes sent */
			return (EIF_INTEGER) ovlp.InternalHigh;
		} else {
			return (EIF_INTEGER) -1;
		}
		FreeLibrary(dllHandle);
	}
```

i.e change `LPTRANSMIT_FILE_BUFFERS` with `void*`
and `TF_USE_KERNEL_APC` with 0x20

This is compiling and executing fine.
But this sounds more a hack than a clean solution.

I will check which version of VS include MSWSock.h